lib/middleware.ts

```diff
@@ -113,6 +113,97 @@ var defaultHandlers = [
   () => ({title: "request-finished"}),
 ];
 
+const defaultContextHandlers = [
+  (req) => ({"request-id": req.headers["x-request-id"]}),
+  (req) => ({"session-id": req.sessionID}),
```

Comment:
@agclever can we remove them for now and merge this. I'm guessing the `unique-id` will be logged by default for all apps?